lib/jekyll/algolia/configurator.rb

```diff
@@ -103,7 +103,7 @@ def self.get(key)
       # Returns the value of this option, or the default value
       def self.algolia(key)
         config = get('algolia') || {}
-        value = config[key] || ALGOLIA_DEFAULTS[key]
+        value = config[key].nil? ? ALGOLIA_DEFAULTS[key] : config[key]
```

Comment:
why changing this to a ternary operator? It now feels weird next to the same pattern used just before. It's very ruby to do `a = b || c`

Reply:
> If || is used it will fall back for `false` to the default value no?

Oh OK OK, now I get it! Yes you're right!